docker-compose.prod.yml

```diff
@@ -25,6 +25,8 @@ services:
       - POSTGRES_USER=${POSTGRES_USER:?required}
       - POSTGRES_PASSWORD=${POSTGRES_PASSWORD:?required}
       - FORTYTWO_APP_CALLBACK_URL=${FORTYTWO_APP_CALLBACK_URL:?required}
+      - WEBSITE_OWNER_PASSWORD=123
```

Comment:
probably we don't want this here, but still, I'm leaving it as a placeholder to discuss it with the reviewers